skyportal/tests/api/test_gcn.py

```diff
@@ -0,0 +1,48 @@
+import os
+import numpy as np
+from astropy.time import Time
+
+from skyportal.models import GcnEvent, Localization
+from skyportal.tests import api
+
+
+def test_gcn(super_admin_token):
+
+    datafile = f'{os.path.dirname(__file__)}/../data/GW190425_initial.xml'
+    with open(datafile, 'rb') as fid:
+        payload = fid.read()
+    data = {'xml': payload}
+
+    status, data = api('PUT', 'gcn/upload', data=data, token=super_admin_token)
+    assert status == 200
+    assert data['status'] == 'success'
+
+    dateobs = Time("2019-04-25 08:18:05", format='iso').datetime
+
+    event = GcnEvent.query.filter_by(dateobs=dateobs).one()
```

Comment:
Instead of querying the database directly to make sure PUTs are successful, the preferred approach is to use the corresponding GET handlers to check these assertions. That way we test those GET handlers as well, and keep the test code separate from underlying database logic by interacting with the test SkyPortal only through the API

Reply:
Yes definitely. Fixed.